index.html

```diff
@@ -859,20 +894,23 @@ <h3>Presentations</h3>
         </p>
         <p>
 <a href="#info-graph-vp"></a> below shows a more complete depiction of a
-[=verifiable presentation=], which is normally composed of at least four
-information [=graphs=]. The first of these [=graphs=], the
-[=verifiable presentation graph=] (which is the [=default graph=]),
-expresses the [=verifiable presentation=] itself, and contains presentation
-metadata. The `verifiableCredential` property in the <a>verifiable
-presentation graph</a> refers to one or more [=verifiable credentials=], each
-being one of the second information [=graphs=], i.e., a self-contained
-[=verifiable credential graph=] which in turn contains credential metadata
-and other claims. Each of these graphs are separate [=named graphs=]. The
-third information [=graph=], the verifiable credential [=proof graph=],
-expresses the credential graph proof, which is usually a digital signature. The
-fourth information [=named graph=], the presentation [=proof graph=],
-expresses the presentation's digital proof, which is usually a digital
-signature.
+[=verifiable presentation=] using an <a>embedded proof</a>
+based on [[?VC-DATA-INTEGRITY]].
+It is composed of at least four information [=graphs=].
+The first of these information [=graphs=], the [=verifiable presentation graph=]
+(which is the [=default graph=]), expresses the [=verifiable presentation=]
+itself through [=presentation=] metadata.
+The [=verifiable presentation=] refers, via the <code>verifiableCredential</code> property,
+to a [=verifiable credential=].
+This credential is a self-contained [=verifiable credential graph=] containing [=credential=] metadata and other [=claims=].
+This [=credential=] refers to a [=verifiable credential=] [=proof graph=] via a <code>proof</code> property,
+expressing the proof (usually a digital signature) of the [=credential=].
+This [=verifiable credential graph=], and its linked [=proof graph=], constitute
+the second and third information [=graphs=], respectively, and are both separate [=named graphs=].
+The [=presentation=] also refers, via the <code>proof</code> property, to
+the [=presentation=]'s [=proof graph=], which is the fourth information [=named graph=].
+This [=presentation=] [=proof graph=] represents the digital signature of the [=verifiable presentation graph=],
+the [=credential graph=], and the [=proof graph=] linked from the [=credential graph=].
```

Comment:
Given that a presentation may contain multiple credentials (and credential proofs), it seems like --
```
the [=credential graph=], and the [=proof graph=] linked from the [=credential graph=]
```
-- should become (something like) --
```
each presented [=credential graph=], and each [=proof graph=] linked from each [=credential graph=]
```
But...
as I understand it, the `[=presentation=] [=proof graph=] represents the digital signature` of the *amalgam* of `the [=verifiable presentation graph=],
the presented [=credential graph(s)=], and the [=proof graph(s)=] linked from the presented [=credential graph(s)=].`

The above may be enough for someone else to finish massaging this sentence, or I can do that after confirmation/correction of my understanding.

Reply:
To simplify the explanation, I decided to concentrate on the case where there is _one_ credential in the presentation, which means that the singular case is o.k. imho. To refer to the case where there are several credentials, I have expanded on the note at the end of the presentation section. See also the comments (https://github.com/w3c/vc-data-model/pull/1404#pullrequestreview-1797263167) on that note.

Reply:
I'm concerned that the multiple VC VP is likely to be more frequently used, and lacking a clear example of this case, it is easy to mess up, particularly in that the `[=presentation=] [=proof graph=]` is not the digital signature of any _portion_ of the VP, but only of the *entire* VP.

I think it's easier to derive a single VC VP implementation from a multiple VC VP example, than it is to derive a multiple VC VP implementation from a single VC VP example, so if we're only going to present one example, I think that it should be a multiple VC VP example.

Reply:
@TallTed,
> I'm concerned that the multiple VC VP is likely to be more frequently used,

I cannot judge that. I would like to get an opinion on that from domain experts like @msporny, @andresuribe87, @dlongley, or @decentralgabe.

> I think it's easier to derive a single VC VP implementation from a multiple VC VP example, than it is to derive a multiple VC VP implementation from a single VC VP example, so if we're only going to present one example, I think that it should be a multiple VC VP example.

I am not sure. Changing the text is easy (sort of), but that would require a change of the diagrams (both the DI and JWT cases) because the current ones are addressing the single VC case only. However, I am afraid that the diagrams would then become huge and would distort the document (the diagrams, as they are, are already fairly large). They would also become fairly complex.

I am also not convinced that such a complicated diagram (as the "only" diagram) would still be easy to comprehend. I actually think it would become too complicated to the first-time-reader. Note also that none of the examples in the spec contain several credentials explicitly (although, for some examples, there are hints to the possible presence of other credentials.)

What we may do (but I would prefer to do this in a separate PR once this one is, hopefully, solved) is

- Create a separate diagram (or diagrams, including the JWT case) in SVG but ***not*** embedded into this document
- Link to that diagram from the note at the end of the session that addresses the multiple VP case. Actually, I wonder whether it is not better to use that text as a main part of the section (which is non-normative anyway) instead of putting this as a note.

(An alternative might be to come up with a visual clue on the diagrams that provide _hints_ to the possible presence of other credentials and related proofs, but at this moment I have no idea how that could be done properly.)

Reply:
Visual cues (which could be as simple as staggered, stacked replication of the relevant nodes with `...` labels on them) on the existing diagram would help, but I'm less concerned with readers understanding the *idea* of a multiple VC VP than in them understanding the *technicalities* thereof. 

Since we're producing an HTML doc (even though it may be printed or written to a PDF), we don't have to worry so much about page real estate nor about an image being illegible in its primary (possibly scaled down) presentation, as long as it links to a legible presentation (possibly without scaling), so I'm not concerned about a large or complex image. Complexity usually becomes comprehensible by making the image "as large as it needs to be, with as much whitespace as needed to be clear", as opposed to "as small as possible, with as little whitespace as possible, to make it fit on the page."

Given that these are informative, non-normative, we could also consider putting either or both of these diagrams and descriptions into an appendix, to which we link from the main text. I would *not* want to put either diagram or description into something entirely removed from this spec.

Reply:
@TallTed I am willing to give a try working on the diagram to see if it is doable reasonably. But I am still not convinced about your initial statement

> I'm concerned that the multiple VC VP is likely to be more frequently used,

Also, I am still afraid that not only the diagram but also the text would become, in fact, more convoluted and difficult to comprehend if we make this change than leaving it as is.

I would need others to chime in on this.

Reply:
I do not believe it's necessary to have a diagram with multiple credentials in a presentation.